ext/doc/Cassandra/BatchStatement.php

```diff
@@ -56,6 +58,7 @@ public function __construct($type) { }
      *
      * @return \Cassandra\BatchStatement self
      */
-    public function add($statement, $arguments) { }
-
+    public function add($statement, $arguments = null): \Cassandra\BatchStatement
```

Comment:
You could've used `: self` for quite a lot of these

Reply:
I didn't manually edit these files, everything is generated (already told you that)